components/infobox/wikis/stormgate/infobox_unit_custom.lua

```diff
@@ -259,11 +282,13 @@ function CustomUnit._hotkeys(hotkey1, hotkey2)
 end
 
 ---@param inputString string?
----@return string[]
+---@return string
 function CustomUnit:_displayCommaSeparatedString(inputString)
```

Comment:
Should we do here also the `.csvToPageList` and the `._displayCsvAsPageCsv` (although we don't have any Cells at the moment that only show `.csvToPageList` - all would be converted to `._displayCsvAsPageCsv` )? - I gues it woudl still be better if we have it consistent.

Same question would arise for `extensions_attack`

Reply:
Edit: can you also please check the Annotations of the two csv functions. Not sure if table or string[] is correct there - what does string[] exactly mean?

Reply:
- having it in 1 function is fine, in the other module it was just to deduplicate code
- `string[]` means an array of strings, e.g. `{string1, string2, string3}`